custom_dropdown.html

```diff
@@ -38,11 +38,12 @@
           };
 
           $('#st-search-input').swiftype({
-            engineKey: 'V7U1B7opm5xDnVFnsk5d',
+            engineKey: '5jZG1gmmCTFYbSSDjpqq',
```

Comment:
@goodroot Is this engine key supposed to be in here? Just saw that it changed, wanted to make sure it was intentional.

Reply:
@JasonStoltz Yes -- the prior key is a... strange key.